colorcoder/ColorPair.java

```diff
@@ -0,0 +1,33 @@
+package colorcoder;
+public class ColorPair {
+	final static String MajorColorNames[] = {
```

Comment:
indentation is required for clear readability.

Reply:
changes done

Reply:
👍